packages/test/test-end-to-end-tests/src/test/stashedOps.spec.ts

```diff
@@ -969,6 +970,75 @@ describeCompat("stashed ops", "NoCompat", (getTestObjectProvider, apis) => {
 		assert.strictEqual(map2.get(testKey), testValue);
 	});
 
+	it.skip("ensure single attach op sent in map", async function () {
+		let idB;
+		const cb = async (container, d?) => {
+			const defaultDataStore = (await container.getEntryPoint()) as ITestFluidObject;
+			const runtime = defaultDataStore.context.containerRuntime;
+
+			const dataStoreB = await runtime.createDataStore(["default"]);
```

Comment:
I would suggest using a simpler container schema / default data store, to be more explicit in the test code itself about what's going on.  IIRC, the key part of the setup is that dataStoreB has internal handled between its DDSes (in a Map).  Better to have that be explicit in the test code itself.

Probably will get here anyway by following Tony's suggestions below.

Reply:
This will probably be resolved in a follow up, just so we can have some coverage immediately.